README.md

```diff
@@ -7,56 +7,190 @@ Lisk is a next generation crypto-currency and decentralized application platform
 [![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](http://www.gnu.org/licenses/gpl-3.0)
 [![Join the chat at https://gitter.im/LiskHQ/lisk](https://badges.gitter.im/LiskHQ/lisk.svg)](https://gitter.im/LiskHQ/lisk?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-**NOTE:** The following information is applicable to: **Ubuntu 14.04, 16.04 (LTS) or 16.10 - x86_64**.
-
 ## Prerequisites - In order
 
+This sections provides details on what you need install on your system in order to run Lisk.
+
+### System Install
+
 - Tool chain components -- Used for compiling dependencies
 
-  `sudo apt-get install -y python build-essential curl automake autoconf libtool`
+    - Linux:
```

Comment:
It is worth to mention that apt-get tool is present on Debian and Ubuntu only.
Otherwise replace "Linux" with "Ubuntu/Debian".

Reply:
Done 👍